ChildProject/pipelines/metrics.py

```diff
@@ -26,7 +27,7 @@ class Metrics(ABC):
     :type project: ChildProject.projects.ChildProject
     :param metrics_list: pandas DataFrame containing the desired metrics (metrics functions are in metricsFunctions.py)
     :type metrics_list: pd.DataFrame
-    :param by: unit to extract metric from (recording_filename, experiment, child_id, session_id), defaults to 'recording_filename'
+    :param by: unit to extract metric from (recording_filename, experiment, child_id, session_id, segments), defaults to 'recording_filename', use 'segments' to use the segments argument
```

Comment:
should we have a different explanation for segments here? perhaps something like: "'segments' is mandatory if passing the segments (-s) argument"

Reply:
We don't include info about CLI arguments (--segments -d) because this is the docstring of the function, not the help of the CLI